### PR TITLE
feat: worklfow dispatch

### PR DESCRIPTION
**Pull request by uoboda-splunk**

### CODE REVIEW

File: .github/workflows/sync.yaml [hunk 1]
```diff
@@ -52,14 +76,16 @@ jobs:
           git config --global user.email ${{ secrets.SA_GH_USER_EMAIL }}
           git config --global user.name ${{ secrets.SA_GH_USER_ADMIN }}
           git config --global commit.gpgsign true
-          python3 tools/sync.py tools/sync.sh repositories_main.csv
+          python3 tools/sync.py tools/sync.sh repositories_main.csv -mr ${{ github.event.inputs.repositories }}
```
Comment (artemrys) [inline .github/workflows/sync.yaml:79]: Honestly, I am not sure that we need to have 2 different parameters for the repositories to sync. Both those params server the same purpose. I think we don't need to provide `repositories_main.csv` parameter.

Comment (uoboda-splunk) [reply .github/workflows/sync.yaml]: Yes, this parameter is rather useless. Deleted

File: tools/inputs_validator.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def validate_repositories(repos: str) -> None:
+    """
+    Validates repositories input value
+    """
+    if not repos:
+        __raise_error("repos input cannot be empty")
+    elif repos == ALL_SYNC_REPOS_VALUE:
+        if not os.getenv("GITHUB_ACTOR") in POWER_USERS:
```
Comment (artemrys) [inline tools/inputs_validator.py:16]: Nice!

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,5 @@ work
 !enforce/.*
 .env
 .DS_Store
+*__pycache__*
+gd
```
[the remark this answers is not in the document]
Comment (uoboda-splunk) [reply .gitignore]: typo :) deleted

**Review by artemrys — APPROVED** [review]

**Pull request merged**